gui-apps/swaync/files/swaync-0.8.0-pulsefree.patch

```diff
@@ -0,0 +1,942 @@
+diff --git a/.github/workflows/arch-build.yml b/.github/workflows/arch-build.yml
```

Comment:
There's no link to where this is from or explaining what it does (or why). Please add a link and some explanation.

This patch is especially large and I'm not a fan of carrying this indefinitely if there's no PR pending upstream either.